src/main/java/mate/academy/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities, find and read articles about problems
Lombok has a lot of other annotations that replace boilerplate code
Don't forget to add Table annotation, to specify the name of table according to the naming convention